geopatterns_demo/file_handler.py

```diff
@@ -0,0 +1,74 @@
+from io import BytesIO
+
+import boto3
+
+from geopatterns_demo import models, settings
+from geopatterns_demo.draw import geopattern
+
+
+def hash_name(text: str) -> str:
+    """Generate the file name using the hash function.
+
+    Args:
+        text (str): text that we get from get request
+
+    Returns:
+        str: file name
+    """
+    return 'image{0}.png'.format(hash(text))
+
+
+def check_file(file_name: str, bucket: boto3) -> bool:
+    """Check the file for presence in the S3 bucket.
+
+    Args:
+        file_name (str): image file name
+        bucket (boto3): boto3 object - S3 bucket
+
+    Returns:
+        bool: bool
+    """
+    bucket_object = list(bucket.objects.filter(Prefix=file_name))
```

Comment:
This method of finding an object on a bucket is probably not optimal. A proper method would be using, say, this: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.head_object

But in this particular case I believe we should not do it at all. See below.